script/testing/oltpbench/README.md

```diff
@@ -0,0 +1,65 @@
+# OLTP Benchmark Testing
+This folder include the necessary files for running oltpbench testings.
+
```

Comment:
We need a "how to run" section which should probably go before what it does.

Reply:
I think the two main goals of this documentation is 
1)to describe how would someone execute a test
2) How would someone reuse a component if it fits their need (AKA. They want to run an OLTPBench test as part of another script.)
A secondary goal, which I think you've done a good job of addressing, is what do people need to pay attention to if they are making a change.

Reply:
For the first goal, I added the `How to run it?` section at the beginning of the document